common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -290,9 +294,19 @@ def test_deferred_verification_enrollment(self):
         enrollment_mode = self.dashboard_page.get_enrollment_mode(self.course_info["display_name"])
         self.assertEqual(enrollment_mode, 'verified')
 
+        # Expect enrollment activated event
+        self.assertEqual(
```

Comment:
Thanks for for making this change.  It looks great to me!  I agree that making this logic into a helper would be useful in the future, but also that it can be done in a followup PR if you prefer.